src/main.cpp

```diff
@@ -49,8 +49,13 @@ void loadTranslations(QTranslator *translator, const QString &locale) {
     loadTranslation(translator[5], appPath + "/languages/QOwnNotes_" + locale);
     loadTranslation(translator[6], appPath + "/QOwnNotes_" + locale);
     loadTranslation(translator[7], "../src/languages/QOwnNotes_" + locale);
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
```

Comment:
You'll need a `>=` check here, wouldn't you?

Reply:
indeed, fixed.

Reply:
thank you